projects/components/src/toggle-switch/toggle-switch.component.ts

```diff
@@ -26,6 +27,9 @@ export class ToggleSwitchComponent {
   @Input()
   public disabled?: boolean;
 
+  @Input()
+  public size?: string = 'small';
```

Comment:
So default it "small". Do we know we still have to support medium(now or in future)?

Also, if we are taking size as input, better to use `const enum` as we can type it to the two specific options. Example in Icon component.

Reply:
Since the small use case is new, I'd say lets default to medium and use small as the override.

Reply:
Ok. I've just set medium as default and small as the new case. I also added the enum.